server.js

```diff
@@ -1,7 +1,26 @@
-const app = require('./src/app');
+const express = require ("express");
+const { contentType } = require("express/lib/response");
+const app = express();
 
+app.use(express.json());
 
-const PORT = process.env.PORT || 3000;
-app.listen(PORT, () => {
-  console.log(`Server is live on port ${PORT}.`);
-});
+app.post( "/register",(req, res)=>{
+    var nameformat = (/^[A-Za-z\s]*$/)
+    var mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+    var passformat=  /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[^a-zA-Z0-9])(?!.*\s).{8,15}$/;
+    var phoneno = /^\d{11}$/
+    if(req.body.name.match(nameformat) && 
```

Comment:
es mn her dafa req.body likh rhy ho hr property k sath... aik kam karo if se pehly ye likho 

`const { name, email, password, phoneNumber } = req.body;`

phir if mn direct variable use kr lo like this:

`if (name.match(nameformat)`